interface-definitions/interfaces-bridge.xml.in

```diff
@@ -126,6 +126,45 @@
                   </completionHelp>
                 </properties>
                 <children>
+                  <leafNode name="native-vlan">
+                    <properties>
+                      <help>Set the interface in ACCESS mode, automatically append vlan id to packets without the vlan id  in the incoming traffic, and automatically strip the vlan id when exporting packets with specific vlan id</help>
+                      <completionHelp>
+                        <list>disable</list>
```

Comment:
Why do I need to disable the native VLAN? When I do not define it it should be disabled. We should not add superfluous CLI nodes.

Reply:
Because in the bridge with vlan_filter enabled by default, each port has a vlan 1 by default, `disable` will mean that a port will neither set `native-vlan` nor delete vlan 1

Reply:
This might be the underlaying implementation, I'm referring to the CLI. If there is no `native-vlan` node defined, it corresponds to `disable` thus you can get rid of the entire `disable` node.

Reply:
OK, I have updated the PR